truss/templates/server/common/truss_server.py

```diff
@@ -255,6 +256,20 @@ def create_application(self):
             },
         )
 
+        async def dispatch(request, call_next):
```

Comment:
why is this called `dispatch`? Could we name this middleware something more meaningful?